<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>植物工厂</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/back.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>植物品种</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PortBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Plant_varieties">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>玫瑰</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>香菇</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>模式选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Model_selection">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>自动</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>手动</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="OpenSerialButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#OpenSerialButton
{
    border-image: url(:/image/button.png);
}


QPushButton#OpenSerialButton:pressed
{
  
	border-image: url(:/image/button1.png);
}</string>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>40</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>0</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LcdTemp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TemplineEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TemplineEdit_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LcdHumi">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TemplineEdit_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>光照</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LcdLight">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BntMotor_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#BntMotor_2
{
    border-image: url(:/image/button.png);
}


QPushButton#BntMotor_2:pressed
{
  
	border-image: url(:/image/button1.png);
}</string>
    </property>
    <property name="text">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TemplineEdit_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TemplineEdit_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lx</string>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_3{
     border-image: url(:/image/button.png);
     border-style: outset; /*边框样式*/
     border-width: 4px;  /*边框宽度为4像素*/
     border-radius: 10px; /*边框圆角半径*/
     border-color: rgba(255, 225, 255, 30);/*边框颜色*/
     font: bold 14px &quot;叶根友毛笔行书2.0版&quot;;/*字体*/
     color:rgba(0, 0, 0, 100);/*字体颜色*/
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="PwmSlider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>97</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="PwmLineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>95</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>95</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>2</width>
      <height>2</height>
     </size>
    </property>
    <property name="text">
     <string>调节光照</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ColdButton">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>45</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ColdButton
{
    border-image: url(:/image/button.png);
}


QPushButton#ColdButton:pressed
{
  
	border-image: url(:/image/button1.png);
}</string>
    </property>
    <property name="text">
     <string>制冷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WindBotton">
    <property name="geometry">
     <rect>
      <x>692</x>
      <y>46</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#WindBotton
{
    border-image: url(:/image/button.png);
}


QPushButton#WindBotton:pressed
{
  
	border-image: url(:/image/button1.png);
}</string>
    </property>
    <property name="text">
     <string>通风</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WetButton">
    <property name="geometry">
     <rect>
      <x>692</x>
      <y>3</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#WetButton
{
    border-image: url(:/image/button.png);
}


QPushButton#WetButton:pressed
{
  
	border-image: url(:/image/button1.png);
}</string>
    </property>
    <property name="text">
     <string>加湿</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HotButton">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>3</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#HotButton
{
    border-image: url(:/image/button.png);
}


QPushButton#HotButton:pressed
{
  
	border-image: url(:/image/button1.png);
}</string>
    </property>
    <property name="text">
     <string>加热</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
